### PR TITLE
SorTask903 Unit test convert_seconds_to_pandas_minutes() 

### PR DESCRIPTION
**Pull request by neha2801-create**
#903 
There is a convert_seconds_to_pandas_minutes() in helpers/hdatetime.py

Added 2 unit tests to helpers/test/test_hdatetime.py:

With input values :
 input = 300
 input = 10

### CODE REVIEW

File: helpers/test/test_hdatetime.py [hunk 1]
```diff
@@ -576,6 +576,39 @@ def test3(self) -> None:
...
 
+# #############################################################################
+# Test_convert_seconds_to_minutes
+# #############################################################################
+
+
+class Test_convert_seconds_to_minutes(hunitest.TestCase):
+    def test1(self) -> None:
+        """
+        Test case for input which is an integer number of minutes.
```
Comment (DanilYachmenev) [inline helpers/test/test_hdatetime.py:587]: -> `Check that conversion is implemented correcty.`

File: helpers/test/test_hdatetime.py [hunk 2]
```diff
@@ -576,6 +576,39 @@ def test3(self) -> None:
...
+        """
+        num_secs = 300
+        act = hdateti.convert_seconds_to_minutes(num_secs)
+        exp = int(num_secs / 60)
+        self.assert_equal(str(act), str(exp), fuzzy_match=True)
+
+    def test2(self) -> None:
+        """
+        Test case for input which is not an integer number of minutes.
```
Comment (DanilYachmenev) [inline helpers/test/test_hdatetime.py:596]: -> `Check that an error is raised when input is not an integer number of minutes.`

File: helpers/test/test_hdatetime.py [hunk 3]
```diff
@@ -576,6 +576,39 @@ def test3(self) -> None:
...
+
+class Test_convert_seconds_to_minutes(hunitest.TestCase):
+    def test1(self) -> None:
+        """
+        Test case for input which is an integer number of minutes.
+        """
+        num_secs = 300
+        act = hdateti.convert_seconds_to_minutes(num_secs)
+        exp = int(num_secs / 60)
+        self.assert_equal(str(act), str(exp), fuzzy_match=True)
```
Comment (DanilYachmenev) [inline helpers/test/test_hdatetime.py:592]: you can use -> `self.assertEqual(act, exp)` here in order to compare int values

`assert_equal()` is indeed for string values

ofc double-check that everything passes

**Review by DanilYachmenev — COMMENTED** [review]
neat!
couple nits and merge

**Comment by neha2801-create** [comment]
Hi @DanilYachmenev ,
Can you please review my request, without that I cannot merge?

**Review by DanilYachmenev — COMMENTED** [review]
lgtm, merging to proceed
@samarth9008

**Review by DanilYachmenev — APPROVED** [review]

**Pull request merged**